convert2rhel/actions/post_conversion/remove_tmp_dir.py

```diff
@@ -0,0 +1,59 @@
+# Copyright(C) 2024 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import errno
+import logging
+import shutil
+
+from convert2rhel import actions
+from convert2rhel.utils import TMP_DIR
+
+
+loggerinst = logging.getLogger(__name__)
+
+
+class RemoveTmpDir(actions.Action):
+    id = "REMOVE_TMP_DIR"
+    dependencies = ()
```

Comment:
```suggestion
    dependencies = ("UPDATE_GRUB",)
```
This is why the integration tests are failing. The `UPDATE_GRUB` makes backup of some boot files (some extra safety) and creates the backup in the `TMP_DIR` path